src/etc/inc/services.inc

```diff
@@ -1748,7 +1748,7 @@ EOD;
                 $need_update = false;
 
                 /* Update IPv4 if we have it. */
-                if (is_ipaddrv4($wanip)) {
+                if (is_ipaddrv4($wanip) && $dnsupdate['recordtype'] != "AAAA") {
```

Comment:
Well I see yout point. Do you think, we should rename "both" to "ALL" which represents all normal DNS records as A, AAAA (and maybe AAAAAAAA in some years)?

Any other record type has to be set by an extra configuration option as MX, right?